test/built-ins/Proxy/has/return-true-without-same-target-prop.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.7
+description: >
+    The result of [[HasProperty]] is a Boolean value and will affect has
+    checkings. True returned when target property doesn't exists;
+---*/
+
+var p = new Proxy({}, {
+    has: function(t, prop) {
+        return true;
+    }
+});
+
+assert.sameValue(('attr' in p), true);
+
+with (p) {
+    assert.sameValue(attr, undefined);
```

Comment:
Seems like the 'has' trap reporting true always will mean assert itself will attempt to be looked up on the with object?